styles/facts.css

```diff
@@ -0,0 +1,148 @@
+@import url('https://fonts.googleapis.com/css2?family=Felt+Marker&display=swap');
+
+
+#fact-body {
+  margin: 0;
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  background-image: url("../assets/AwesomeFieriPic.png");
+  background-repeat: repeat;
+}
+.fact-container {
```

Comment:
I suggest keeping a single blank line between each CSS rule to make it easy to see at a glance where one rule set and an another begins.